themes/vocabulary_theme/templates/layout.html

```diff
@@ -73,12 +71,13 @@
                 ['/contributing-code/github-repo-guidelines', 'GitHub Repo Guidelines'],
                 ['/contributing-code/cc-search', 'CC Search'],
               ] %}
-                <a class="navbar-item" href="{{ href|url }}">{{ title }}</a>
+              <a class="navbar-item" href="{{ href|url }}">{{ title }}</a>
```

Comment:
The indentation was correct. You should indent the HTML that lies inside loops or conditions.